src/controllers/order.conrtoller.ts

```diff
@@ -5,6 +5,7 @@ import { getRepository, Repository } from 'typeorm'
 import Order from '../models/order.entity'
 import Dish from '../models/menu.entity'
 import CartPosition from '../models/cart.entity'
+import TimetobookEntity from "../models/timetobook.entity";
```

Comment:
Again, somewhere semicolons, somewhere not. Lets check your files again, and take care of eslint